backend/config/runtime.exs

```diff
@@ -65,11 +65,6 @@ if config_env() == :prod || config_env() == :staging do
       ip: {0, 0, 0, 0, 0, 0, 0, 0},
       port: port
     ],
-    # The secret key base is used to sign/encrypt cookies and other secrets.
```

Comment:
Why removing this comment?
Could be helpful to have an explanation on what this key is used for... Maybe shorter ^^

Reply:
yep ;)